lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
```

Comment:
@patrickhulce do you want to craft those changes in a PR? I am working on some other stuff with `<template>` and can rebase when that lands.